tools/clang/lib/Sema/SemaOverload.cpp

```diff
@@ -10936,7 +10936,8 @@ bool Sema::buildOverloadedCallSet(Scope *S, Expr *Fn,
     // We don't perform ADL for implicit declarations of builtins.
     // Verify that this was correctly set up.
     FunctionDecl *F;
-    if (ULE->decls_begin() + 1 == ULE->decls_end() &&
+    if ((ULE->getNumDecls() > 0) &&
```

Comment:
nit: Can you wrap this in a comment block to denote that it is a `// HLSL change`?

Reply:
Done.